### ISSUE TITLE
AI planning for sweep missions

### ISSUE THREAD

**Issue by DanAlbert** [dcs-liberation/dcs_liberation#380]
The AI planner won't plan sweep missions. We'll need to think about when the planner should use sweep, escort, or TARCAP. Maybe base the decision based on total enemy CAP aircraft available?

**Comment by foxwxl** [comment]
IMO and my own experience in BMS, if there are OCA missions planning to T/O soon(ie. in 1 hours) and the enemy AA activities are high, then the sweep mission should be considered before OCA package entering into enemy airspace.

**Comment by DanAlbert** [comment]
I don't think we can really do this until we can plan based on SAM locations (we need to avoid sending sweeps straight into a net of SA-10s), so will be done in a later release.
